### DISCUSSION TITLE
Filter starred repos without lists

### DISCUSSION THREAD

**Discussion by sebthom** [community/community#30242] [Other Feature Feedback, Questions, & Ideas]
The new lists feature is extremely useful.

Is there an option to display all starred repositories that are not yet assigned to any list? (I didn't find one) Since I starred a lot of repos before the lists feature was introduced I would like to find all "untagged" stared repos and categorize them too.

Thanks for any advice in advance!

**Comment by radar-blip** [comment]
I think all repos have to be starred to belong to a list. I'm not sure why that is though.
So, to list all unstarred repos, I dont think you can. 😕

**Comment by NapalmCodes** [comment]
Agreed I need an easy way to get stars without a list assignment so I can add one.

**Reply by Gconquer** [reply]
It just creates more confusion

**Reply by NapalmCodes** [reply]
Confusion around what it’s a filter option or a default uncategorized list?

**Reply by Gconquer** [reply]
seen napalm death when was a teenager mega supporting machine head
